chevah_build

```diff
@@ -34,14 +31,39 @@ PYTHON_BUILD_VERSION=2.7.10
 PYSQLITE_VERSION=2.6.3
 SQLITE_VERSION=3.8.1
 
+REDISTRIBUTABLE_VERSION="9.00.30729.6161"
```

Comment:
I'll take care of this; I have a ticket for fixing the Windows build somehow it does not work for @adiroiban.